tests/unit/test_lazy_load_agate.py

```diff
@@ -0,0 +1,31 @@
+import pytest
```

Comment:
Very minor nit, but given this is net new, the preferred import block order should be:
```python
import sys

import pytest

...
```